### DISCUSSION TITLE
Download only original content from Twitter blog

### DISCUSSION THREAD

**Discussion by syang1999** [TumblThreeApp/TumblThree#411] [Q&A (Question & Answer)]
How can I download only original Tweets and exclude replies from a Twitter account? The options under the 'Blog' tab in the settings are all pertinent to Tumblr only and not Twitter. I tried unchecking a couple (quote, conversation, answer, reblogs) but it still downloads Twitter replies.

Thank you in advance.

**Comment by thomas694** [comment]
The option's in the settings are default values for newly created blogs. Afterwards you can only change the options on the blog's detail page.
Here the option "download reblogged posts" does the correct thing. Unchecking it leaves out any reposts.

**Reply by syang1999** [reply]
I deleted the previous index of the blog and re-added it with "download reblogged posts" unchecked but it still downloaded replies

**Reply by thomas694** [reply]
I see. TumblThree doesn't have an option to skip the download of replies. There was no request for it yet.

**Comment by thomas694** [comment]
Replies can be downloaded/skipped from version 2.10.0.
